domains/runtime/auto-id/src/lib.rs

```diff
@@ -109,7 +109,10 @@ pub type Executive = domain_pallet_executive::Executive<
     Runtime,
     AllPalletsWithSystem,
     // TODO: remove once migrations are done
-    pallet_messenger::migrations::VersionCheckedMigrateDomainsV0ToV1<Runtime>,
+    (
+        pallet_messenger::migrations::VersionCheckedMigrateDomainsV0ToV1<Runtime>,
+        domain_pallet_executive::migrations::StorageCheckedMigrateToEventSegments<Runtime>,
+    ),
```

Comment:
Unrelated to this PR, but we don't need any migration for auto-id, since there is not auto-id domain in Taurus and even the auto-id runtime is not registered in Taurus.